ice/analysis.py

```diff
@@ -74,9 +74,10 @@ def single_sanger_analysis(control_path, sample_path, base_outputname, guide, do
     sa.initialize_with(control_path=control_path,
                        edited_path=sample_path,
                        gRNA_sequences=guide,
-                       indel_max_size=15,
+                       indel_max_size=20,
```

Comment:
What was the rational for changing this to 20?

Reply:
We had customer samples that were returning incorrect results (their confirmed deletion size was outside the bounds of edit proposal range).  We should probably increase indel_max_size to 30, but I did a more conservative increase to 20 for now.

Reply:
COM-1115